sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueServiceClient.java

```diff
@@ -2,25 +2,19 @@
 // Licensed under the MIT License.
 package com.azure.storage.queue;
 
+import com.azure.core.http.rest.PagedIterable;
 import com.azure.core.http.rest.Response;
 import com.azure.core.http.rest.SimpleResponse;
 import com.azure.core.http.rest.VoidResponse;
-import com.azure.storage.common.AccountSASPermission;
-import com.azure.storage.common.AccountSASResourceType;
-import com.azure.storage.common.AccountSASService;
-import com.azure.storage.common.IPRange;
-import com.azure.storage.common.SASProtocol;
+import com.azure.storage.common.*;
```

Comment:
Don't have `*` imports. (This is likely an IntelliJ configuration to squash imports to a `*` after 5 imports from the same package)